public/src/components/ToastManager.jsx

```diff
@@ -0,0 +1,198 @@
+import { useState, useRef } from 'react';
+import * as React from 'react';
```

Comment:
These can just be

import React, { useState, useRef} from 'react';

Reply:
Fixed